src/java/org/apache/cassandra/db/compaction/CompactionManager.java

```diff
@@ -1520,6 +1527,16 @@ static void compactionRateLimiterAcquire(RateLimiter limiter, long bytesScanned,
         }
     }
 
+    private long getLengthRead(long bytesScanned, long lastBytesScanned, double compressionRatio)
```

Comment:
@maoling do we need these two methods? they are just oneliners and not used anywhere else. Just inline it into `compactionRateLimiterAcquire` I guess?